metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_helper.py

```diff
@@ -15,9 +15,15 @@ def unquote_and_decode_unicode_escape_seq(
     if string.startswith(leading_quote) and string.endswith(trailing_quote):
         string = string[1:-1]
 
-    cleaned_string = string.encode().decode("unicode-escape")
-
-    return cleaned_string
+    # Decode Unicode escape sequences. This avoid issues with encoding
+    while string.find("\\u") >= 0:
+        index = string.find("\\u")  # The first occurrence of the substring
+        unicode_seq = string[index : (index + 6)]  # The Unicode escape sequence
```

Comment:
the 6 = 2 from `\u` and 4 from the 4 hex digits, right?

what happens if someone actually has a `\u` in their description?

Reply:
I believe it will identify it as a sequence of unicode characters. I can add an additional logic to confirm that the next 4 digits are a valid hex sequence. Maybe this will prevent this problem.

Reply:
I runned a lot of different inputs and i think i found a more robust version of the utility function. What do you think?